arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
```

Comment:
@taiwu Yes please. Follow the guidelines outlined https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md#filenames-and-folder-structure .

Please make sure you create 2 commits. One with just copy of the old swagger into the new folder and second with the changes to help us review the changes only. Thanks.